docs/synthetics/faq.md

```diff
@@ -30,7 +30,7 @@ If you do not want to run private agents yourself, you can also configure the fi
 No. The HTTP monitor does not support OAuth authentication before sending the actual request. HTTP monitor supports header-based authentication. 
 
 ### Can I monitor API endpoints protected by Basic authentication?
-Yes, the HTTP monitor supports header-based authentication. Custom HTTP request headers can be specified when creating a monitor.  For Basic authentication add the `Authorization: Basic XYZ` header where XYZ is the Base64 encoding of the username:password string.
```

Comment:
A person who may be reading this and is using Browser monitor might be mislead because they will not see "HTTP monitor" here.

Reply:
OK, I will re-add a short sentence highlighting this.